server/templates/custom_dc/unsdg/homepage.html

```diff
@@ -0,0 +1,72 @@
+{#
+ Copyright 2023 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+#}
+{%- extends BASE_HTML -%}
+
+{% set main_id = 'homepage' %}
+{% set page_id = 'page-homepage' %}
+{% set title = 'Home' %}
+{% set is_hide_full_footer = true %}
+{% set is_hide_sub_footer = true %}
+
+{% block head %}
+<link rel="shortcut icon" href="http://un.org/favicon.ico" />
+<link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['VERSION'])}}>
+{% endblock %}
+
+{% block content %}
+<section id="homepage-top">
+  <div class="container">
+    <div class="row">
+      <div class="lead-col col-6 col-lg-6">
+        <img src="/custom_dc/unsdg/poster.png" />
+      </div>
+      <div class="lead-col col-6 col-lg-6">
+        <h3>{% trans %}SDG Indicators for {% endtrans %}</h3>
```

Comment:
nit: Should we reword:

Explore SDG by country
* Mexico
* ...
* 

Explore SDG using natural language
* [Poverty in Africa](...)
* [Health across the world](...)

?

Reply:
Done